filter_plugins/filter_hosts.py

```diff
@@ -1,5 +1,8 @@
-def get_machine_identifier(v):
-    return v['ansible_host'] if 'ansible_host' in v else v['inventory_hostname']
+def get_machine_hostname(v):
+    if 'ansible_host' in v:
+        return v['ansible_host']
+
+    raise Exception(f'Instance {v["inventory_hostname"]} has not "ansible_host" option!')
```

Comment:
`has not` -> `hasn't`? `doesn't have`? `has not`?

I'm not sure it's wrong.

Reply:
I think everything is right